### DISCUSSION TITLE
Issue with displaying repeat events.

### DISCUSSION THREAD

**Discussion by iamradrob** [MMRIZE/MMM-CalendarExt3#133] [Q&A] [answered]
Hello, I was having an issue where repeat events aren't showing up unless its an all day. Any help would be greatly appreciated. Thank you

**Comment by eouia** [comment]
Probably that was the issue of your default calendar module, not this CX3.

**Comment by tetron** [comment, marked as answer]
This is an upstream issue with `rrule` which is used by `node-ical` which used by the MagicMirror calendar fetcher which is used by CalendarExt3.  The immediate solution is to install node-ical 0.17.0.

https://github.com/MagicMirrorOrg/MagicMirror/issues/3351

**Reply by iamradrob** [reply]
Thank you so much, really appreciate the help!
